### DISCUSSION TITLE
Stay signed in forever

### DISCUSSION THREAD

**Discussion by Sesa1988** [supabase/supabase#5093] [Questions]
Hi,

Im not sure about authentication in supabase. Usually you have a max login duration like on Identity Server 4 no matter what and even with refresh tokens if Im not wrong. 

I set the time to one week at the supabase configuration UI. Does this mean that I can call restoresession for one week and after that my user would be signed out?

I dont want my users to be signed out if they doesnt open my app for one week.

**Comment by Aldo111** [comment]
Didja figure this out?

**Reply by Sesa1988** [reply]
I'm not sure. I opened a bug that the refresh token don't refresh when it reach its configured limit. 
I tested it on the dev environment with one hour and it looked fixed but I'm not sure if this means that you can be signed in forever.

**Reply by Aldo111** [reply]
Gotcha. Been trying it out now - so far no issues I think but I guess I'll know for sure with heavy testing!
